### PR TITLE
Remove wsl

### PR DESCRIPTION
**Pull request by mschwoer**
Remove WSL support.

Refactor cli module to become less wordy.

### CODE REVIEW

File: alphadia/cli.py [hunk 1]
```diff
@@ -335,36 +190,38 @@ def run(*args, **kwargs):
...
+    fasta_path_list = _get_fasta_list_from_args_and_config(args, user_config)
     logger.progress(f"Using {len(fasta_path_list)} fasta files:")
     for f in fasta_path_list:
         logger.progress(f"  {f}")
 
     # TODO rename all output_directory, output_folder => output_path, quant_dir->quant_path (except cli parameter)
-    logger.progress(f"Saving output to: {output_directory}")
+    quant_dir = _get_from_args_or_config(
+        args, user_config, args_key="quant_dir", config_key="quant_dir"
+    )
```
Comment (mschwoer) [inline alphadia/cli.py:224]: just some reordering to have the extraction of the value and the log statement close by

File: gui/package-lock.json [hunk 2]
```diff
@@ -13188,17 +13187,6 @@
         "node": ">=0.10.0"
       }
     },
-    "node_modules/is-wsl": {
```
Comment (mschwoer) [inline gui/package-lock.json:13191]: this I am not sure about.. was never references in the code but clearly points to wsl

**Review by anna-charlotte — APPROVED** [review]
looks good to me—just wondering though, why are we removing WSL support? No longer needed? I'm not too familiar with it.

File: alphadia/cli.py [hunk 3]
```diff
@@ -135,113 +114,43 @@ def parse_config(args: argparse.Namespace) -> dict:
...
-
-    raw_path_list : list
-        List of raw files.
-    """
-    config_raw_path_list = config.get("raw_path_list", [])
-    raw_path_list = (
-        utils.windows_to_wsl(config_raw_path_list) if args.wsl else config_raw_path_list
-    )
-    raw_path_list += utils.windows_to_wsl(args.file) if args.wsl else args.file
+    if (config_directory := config.get("directory")) is not None:
```
Comment (GeorgWa) [inline alphadia/cli.py:148]: should we add a `valid_extensions` list?

File: alphadia/cli.py [hunk 4]
```diff
@@ -335,36 +190,38 @@ def run(*args, **kwargs):
...
         return
 
-    quant_dir = parse_quant_dir(args, config)
-
     reporting.init_logging(output_directory)
-    raw_path_list = parse_raw_path_list(args, config)
-
-    library_path = parse_library(args, config)
-    fasta_path_list = parse_fasta(args, config)
+    logger.progress(f"Saving output to: {output_directory}")
```
Comment (GeorgWa) [inline alphadia/cli.py:204]: I think we should also move all logging into the `SearchPlan`

File: gui/src/main/modules/engine.js
Comment (GeorgWa) [inline gui/src/main/modules/engine.js:1]: looks good

**Review by GeorgWa — APPROVED** [review]
LGTM

File: alphadia/cli.py (hunk 3 above)
Comment (mschwoer) [reply alphadia/cli.py:148]: you mean filter out all raw files that are not `.d`, `.raw`, .. ?

**Pull request merged**